src/components/ContentBook.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ContentBook = ({ title, author }) => (
+  <li>
+    <h3>{title}</h3>
+    <p>{author}</p>
+    <button type="button">Delete</button>
+  </li>
+);
```

Comment:
- [ ] Kindly make use of `useDispatch` to dispatch `REMOVE_BOOKS` reducer from your bookSlice to **remove** a book from the `books` array (with the *id* attribute)

Reply:
used dispatch